LICENSE

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2020-202x The virtualenv developers
+Copyright (c) 2022, Oracle and/or its affiliates
```

Comment:
You can't change the license, sorry.

Reply:
Yes, this is meant to be just a copyright, no license change. Related discussion: https://github.com/pyenv/pyenv/pull/1594#discussion_r414980992. In any case, if this is going to be a separate plugin we can defer this to later time

Reply:
You can't change the copyright either.